app/helpers/access_lint_helper.rb

```diff
@@ -0,0 +1,19 @@
+module AccessLintHelper
+  def include_access_lint
+    if Rails.env.production?
+      js_file = "//accesslint.com/1.0/accesslint.js"
+    else
+      js_file = "accesslint"
+    end
+
+    javascript_include_tag(js_file) +
+    javascript_tag(<<-JS.strip_heredoc)
+      window.addEventListener("load", function load(){
+        window.removeEventListener("load", load, false);
```

Comment:
@geoffharcourt this code was inspired by:

https://developer.mozilla.org/en-US/Add-ons/Code_snippets/On_page_load#Attaching_a_script

Also, `load` is properly scoped, and not exposed on `window`. The call to `removeEventListener` accepts the `load` function as an argument and will only remove it. 

This approach is preferred over setting `window.onload` for the sake of not clobbering existing listeners.

> -- @seanpdoyle